source/_includes/doc_snippets/number-units.liquid

```diff
@@ -105,18 +106,9 @@ to get the time it should take.
 
   .slider
     @include move(10px, 120px)
-  ===
-  .slider {
-    position: absolute;
-    left: 10px;
-    transition: left 2.2s;
-  }
-  .slider:hover {
-    left: 120px;
-  }
```

Comment:
In cases where the compiled CSS is now identical to the hardcoded output like this, we removed the hardcoded version.